lib/game.rb

```diff
@@ -1,3 +1,32 @@
 class Game
-    
+    attr_reader :game_id,
+                :season,
+                :type,
+                :away_team_id,
+                :home_team_id,
+                :away_goals,
+                :home_goals
+
+    def initialize(game_id, season, type, away_team_id, home_team_id, away_goals, home_goals)
```

Comment:
Change initialize method to accept a hash argument rather than a series of arguments.  Also update the instance variables to update from the hash.